src/CalculateMethod.php

```diff
@@ -87,6 +87,21 @@ public function getMethods($country_code, $package_value, $package_weight)
     }
 
     /**
+     * Get full list of available RoyalMail Methods
+     *
+     * @return array
+     */
+    public function getAllMethods()
+    {
+        $methods = [];
```

Comment:
We're moving away from long array format for the library because we only want to php 5.4 and up.